cmd/launcher/svc_windows.go

```diff
@@ -51,6 +51,18 @@ func runWindowsSvc(systemSlogger *multislogger.MultiSlogger, args []string) erro
 
 	// Create a local logger. This logs to a known path, and aims to help diagnostics
 	if opts.RootDirectory != "" {
+		// check for old root directories before creating DB in case we've stomped over with windows MSI install
+		updatedRootDirectory := determineRootDirectory(opts.RootDirectory, systemSlogger.Logger)
```

Comment:
I'd use the local/debug slogger, not the system one

Reply:
Though I guess this is happening _before_ we know where to write debug.json. Tricky

Reply:
yeah my first pass at this failed to notice that and the logs end up being split across RootDirs, I thought maybe safer to just do this until we get it set up